packages/react-scripts/config/jest/babelTransform.js

```diff
@@ -10,6 +10,6 @@
 const babelJest = require('babel-jest');
 
 module.exports = babelJest.createTransformer({
-  presets: [require.resolve('babel-preset-react-app')],
+  presets: [[require.resolve('babel-preset-react-app'), { helpers: true }]],
```

Comment:
Doesn't matter for Jest though?

Reply:
Probably not, how about eject? This is why I'd rather it by on by default, see `eject.js`.